lib/models/project.js

```diff
@@ -564,11 +564,14 @@ Project.prototype.generateTestFile = function(/* moduleName, tests */) {
   @private
   @static
   @method closest
-  @param  {String} pathName Path to your project
+  @param  {String} pathName Path to your project (deprecated)
```

Comment:
yes, I think that would make sense because it's the method that's deprecated, not the parameter